docs/strategies/retry.md

```diff
@@ -97,16 +100,16 @@ new ResiliencePipelineBuilder<HttpResponseMessage>().AddRetry(optionsExtractDela
-| `MaxRetryAttempts` | 3                                                                          | The maximum number of retries to use, in addition to the original call.                  |
-| `Delay`            | 2 seconds                                                                  | The base delay between retries.                                                          |
-| `BackoffType`      | Constant                                                                   | The type of the back-off used to generate the retry delay.                               |
-| `UseJitter`        | False                                                                      | Allows adding jitter to retry delays.                                                    |
-| `DelayGenerator`   | `null`                                                                     | Used for generating custom delays for retries.                                           |
-| `OnRetry`          | `null`                                                                     | Action executed when retry occurs.                                                       |
-| `MaxDelay`         | `null`                                                                     | Caps the calculated retry delay to a specified maximum duration.                         |
+| Property           | Default Value                                       | Description                                                                                               |
+|--------------------|-----------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `ShouldHandle`     | Any exceptions except `OperationCanceledException`. | It defines a predicate to determine what results and/or exceptions are handled by the retry strategy.     |
+| `MaxRetryAttempts` | 3                                                   | The maximum number of retry attempts to use, in addition to the original call.                            |
+| `BackoffType`      | Constant                                            | The back-off algorithm type to generate the delay(s) between retry attempts.                              |
+| `Delay`            | 2 seconds                                           | The *base* delay between retry attempts. See the next section for more details.                           |
+| `MaxDelay`         | `null`                                              | If provided then the strategy caps the calculated retry delay to this value.                              |
+| `UseJitter`        | False                                               | If set to `true` a jitter (random value) is added to retry delays. See the next section for more details. |
```

Comment:
Maybe link off to somewhere that explains what jitter is (either our own docs, or say Wikipedia)?

Reply:
Several lines below we have the following explanation:

> `UseJitter`: If enabled, adds a random value between -25% and +25% of the calculated `Delay`, ...

Is this enough?